src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -293,7 +307,7 @@ describe('Detailed view and edit a selected work item - ', () => {
     comp.loggedIn = fakeAuthService.isLoggedIn();
```

Comment:
The fix is in work-item-detail.component.ts and add "public" to workItem: WorkItem;